websites/M/Mavanimes/presence.ts

```diff
@@ -0,0 +1,62 @@
+const mavanimes = new Presence({
+    clientId: "814986239681626143"
+  }),
+ video: {
+  currentTime?: number;
+  duration?: number;
+  paused?: boolean;
+} = {};
+mavanimes.on("iFrameData", (videoData: {
+    currentTime: number;
+    duration: number;
+    paused: boolean;
+  }) => {
+  if (videoData.duration) {
+    video.currentTime = videoData.currentTime;
+    video.duration = videoData.duration;
+    video.paused = videoData.paused;
```

Comment:
video = {...videoData}

Reply:
I don't understand, how should i do that ?